radix-engine/wasm-benchmarks/src/precise_decimal.rs

```diff
@@ -0,0 +1,98 @@
+use radix_engine_common::math::PreciseDecimal;
+
+#[no_mangle]
+pub fn precise_decimal_add(x: i64, y: i64, _cnt: i64) -> i64 {
+    let x = PreciseDecimal::from(x);
+    let y = PreciseDecimal::from(y);
+    let z = x + y;
```

Comment:
Is the `from` cheap, compared with `+`?

Reply:
This is the reason I added `add_no_conversion` benchmark, where I benchmark only `+`.
No, it is not cheap at all. It is exactly the opposite.

Here is the quick comparison
```
                                            + and from      +               calculated from
precise_decimal_add/rust-native/1           137.44 ns       19.405 ns       59 ns
precise_decimal_add/wasmi/1                 60.212 µs       2.4815 µs       29 µs
precise_decimal_add/wasmer/1                2.4495 µs       290.85 ns       1 us
```